.github/workflows/cd.yml

```diff
@@ -188,6 +188,23 @@ jobs:
           git add packages/fx-core/src/common/m365/serviceConstant.ts
           git commit -m "build: replace sideloading placeholders"
 
+      - name: disable chat participant in package.json
```

Comment:
We could also determine whether the extension is prerelease based on fx-core version and vscode-extension version. Is that one more convenient?

Reply:
Confirmed that we also need to remove contributes.chatparticipants in package.json from script.